integration-cli/docker_cli_build_test.go

```diff
@@ -5406,7 +5406,7 @@ func (s *DockerSuite) TestBuildShellUpdatesConfig(c *check.C) {
 
 	buildImageSuccessfully(c, name, build.WithDockerfile(`FROM `+minimalBaseImage()+`
         SHELL ["foo", "-bar"]`))
-	expected := `["foo","-bar","#(nop) ","SHELL [foo -bar]"]`
+	expected := `["foo","-bar","#(nop) SHELL [foo -bar]"]`
```

Comment:
I think most commands are affected, but shell was the only one that tested for the exact string.

I guess since more of the commands used append, I can change to using append instead of concat. That way only `workdir()` and `copy/add` will be affected